libraries/rush-lib/src/logic/CredentialCache.ts

```diff
@@ -38,6 +40,7 @@ export interface ICredentialCacheEntry {
  */
 export interface ICredentialCacheOptions {
   supportEditing: boolean;
+  cacheFilePath?: string;
```

Comment:
```suggestion
  /**
   * If specified, use the specified path instead of the default path of `~/.rush-user/credentials.json`
   */
  cacheFilePath?: string;
```